### PR TITLE
add LSCheckstyle

### PR DESCRIPTION
**Pull request by kauboy26**
This package helps programmers write Java code that conforms to CS1331's coding style.

### CODE REVIEW

**Comment by kauboy26** [comment]
'.no-sublime-package' is defined since the plugin uses the subprocess module to run the Java .jar bundled with the plugin.

**Comment by FichteFoll** [comment]
`ctrl+l` is a default binding (on Linux and Windows at least) and ST doesn't like overriding these with multi-chord bindings too much. The package reviewer doesn't check for these currently. I suggest using a different binding or have the user specify it themselves. A `.sublime-command` file would certainly be useful, however.

**Comment by kauboy26** [comment]
@FichteFoll I changed the binding the plugin was using, and added .sublime-commands file.

**Comment by kauboy26** [comment]
Keybindings have been removed, users need to set them up according to their liking.

File: repository/l.json [hunk 1]
```diff
@@ -1391,6 +1391,17 @@
 			]
 		},
 		{
+			"name": "LSCheckstyle",
+			"details": "https://github.com/kauboy26/LimeStyle",
+			"readme": "https://github.com/kauboy26/LimeStyle/blob/master/README.md",
```
Comment (FichteFoll) [inline repository/l.json]: The URL to the readme must be direct. You can just remove this line and PC will pick up the correct URL regardless.

**Comment by kauboy26** [comment]
Keybindings have been removed.
The .sublime-package is necessary.
"l.json" has been updated (readme line removed).

**Pull request merged**